src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -130,6 +131,47 @@ const WorkflowIOTable = ({ which, inputsOutputs, config, errors, onChange, sugge
   ])
 }
 
+class TextCollapse extends Component {
```

Comment:
Can this just be our existing Collapse?

Reply:
Unfortunately, I don't think so--I tried using the existing Collapse but there are too many deeper aspects to the class I'd need to change to have this desired behavior. Isaac suggested just making a new class.